students/11th/JoeunLee/main.html

```diff
@@ -0,0 +1,227 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<meta charset="utf-8">
+	<meta name="viewport" content="width=device-width">
+	<title>westagram-main</title>
+	<link href="style/common.css" rel="stylesheet" type="text/css" />
+	<link href="style/main.css" rel="stylesheet" type="text/css" />
+</head>
+<body>
+	<div class="navOuterContainer">
+		<div class="navInnerContainer">
+			<div class="navLeft"><img src="img/logo_text.png" alt="instagram"></div>
+            <div class="navMid">
+		 	    <input type="text" placeholder="검색">
+		    </div>
+		    <div class="navRight">
+				<div class="navIcon"><a href=""><img class="appliedMargin" src="https://s3.ap-northeast-2.amazonaws.com/cdn.wecode.co.kr/bearu/profile.png" alt="profile"></a></div>
+				<div class="navIcon"><a href=""><img class="appliedMargin" src="https://s3.ap-northeast-2.amazonaws.com/cdn.wecode.co.kr/bearu/explore.png" alt="explore"></a></div>
+				<div class="navIcon"><a href=""><img class="appliedMargin" src="https://s3.ap-northeast-2.amazonaws.com/cdn.wecode.co.kr/bearu/heart.png" alt="heart"></a></div>			   
+				<div class="navIcon"><a href=""><img class="appliedMargin" src="img/IMG_3607.JPG" alt="my page"></div></a>
+			</div>
+		</div>
+	</div>
+	<main class="feedsContainer">
+		<div class="feedsInnerContainer">
+			<div class="feedsStoryBox">
+				<ul class="feedsStoryList">
+					<li class="feedsStoryListIdBox">
+						<a href=""><img src="img/IMG_5186.JPG"></a>
+						<a href=""><div class="feedsStoryListId">mommy</div></a>
+					</li>
+					<li class="feedsStoryListIdBox">
+						<a href=""><img src="img/weocde.jpeg"></a>
+						<a href=""><div class="feedsStoryListId">wecode</div></a>
+					</li>
+					<li class="feedsStoryListIdBox">
+						<a href=""><img src="img/kim.jpg"></a>
+						<a href=""><div class="feedsStoryListId">glory_K</div></a>
+					</li>
+					<li class="feedsStoryListIdBox">
+						<a href=""><img src="img/IMG_3607.JPG"></a>
+						<a href=""><div class="feedsStoryListId">jjo_nii_</div></a>
+					</li>
+					<li class="feedsStoryListIdBox">
+						<a href=""><img src="img/king.jpg"></a>
+						<a href=""><div class="feedsStoryListId">camellia</div></a>
+					</li>
+					<li class="feedsStoryListIdBox">
+						<a href=""><img src="img/jenny.jpg"></a>
+						<a href=""><div class="feedsStoryListId">jenny.blackk</div></a>
+					</li>
+					<li class="feedsStoryListIdBox">
+						<a href=""><img src="img/gd.jpg"></a>
+						<a href=""><div class="feedsStoryListId">g-dragon</div></a>
+					</li>
+					<li class="feedsStoryListIdBox">
+						<a href=""><img src="img/dua.jpg"></a>
+						<a href=""><div class="feedsStoryListId">dua_lipa_</div></a>
+					</li>
+					<li class="feedsStoryListIdBox">
+						<a href=""><img src="img/wook.png"></a>
```

Comment:
```suggestion
						<a href="">
						  <img src="img/wook.png">
						</a>
```
dom 요소 간 부모 자식 관계가 잘 드러나도록 줄바꿈 해주세요